laiaaa0/assignment2/BinaryTree.cpp

```diff
@@ -65,17 +40,18 @@ BinaryTree<T> * BinaryTree<T>::GetLeftSubtree(){
 
 
 template <class T>
-bool BinaryTree<T>::IsValueInTree (const T & value, BinaryTree<T>* &root_of_value){
+bool BinaryTree<T>::FindValueInTree (const T & value, BinaryTree<T>*& root_of_value){
```

Comment:
Interface-wise, I think is better to change to:
`template <class T>
BinaryTree<T>* BinaryTree<T>::FindValueInTree (const T & value){`
Returning nullptr if not found.